pkg/api/install/install_test.go

```diff
@@ -57,7 +57,7 @@ func TestCodec(t *testing.T) {
 	if err := json.Unmarshal(data, &other); err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	if other.APIVersion != latest.GroupOrDie("").Version || other.Kind != "Pod" {
+	if other.APIVersion != latest.GroupOrDie("").GroupVersion.Version || other.Kind != "Pod" {
```

Comment:
Followup: please note `GroupOrDie` in our issue.  If we know exactly which `GroupVersion` we want, it seems like we should be able to directly depends on its `SchemeGroupVersion`.